### PR TITLE
[#464] Set selection and create copy context menu.

### PR DESCRIPTION
**Pull request by ArneDeutsch**
Signed-off-by: Arne Deutsch <Arne.Deutsch@itemis.de>

### CODE REVIEW

File: org.eclipse.xtext.builder/src/org/eclipse/xtext/builder/trace/TraceEditor.java [hunk 1]
```diff
@@ -109,6 +119,68 @@ public void selectionChanged(SelectionChangedEvent event) {
 				}
 			}
 		});
+		Menu contextMenu = new Menu(text);
+		MenuItem copyItem = new MenuItem(contextMenu, SWT.PUSH);
+		copyItem.setText("&Copy");
+		copyItem.setAccelerator('C');
```
Comment (cdietrich) [inline org.eclipse.xtext.builder/src/org/eclipse/xtext/builder/trace/TraceEditor.java:125]: Bildschirmfoto 2021-02-19 um 09 45 12
Bildschirmfoto 2021-02-19 um 09 43 37
there is something wrong about the shortcut

File: org.eclipse.xtext.builder/src/org/eclipse/xtext/builder/trace/TraceEditor.java [hunk 2]
```diff
@@ -109,6 +119,68 @@ public void selectionChanged(SelectionChangedEvent event) {
 				}
 			}
 		});
+		Menu contextMenu = new Menu(text);
+		MenuItem copyItem = new MenuItem(contextMenu, SWT.PUSH);
+		copyItem.setText("&Copy");
+		copyItem.setAccelerator(SWT.CTRL | 'C');
```
Comment (cdietrich) [inline org.eclipse.xtext.builder/src/org/eclipse/xtext/builder/trace/TraceEditor.java:125]: guess we want cmd on mac

Comment (ArneDeutsch) [reply org.eclipse.xtext.builder/src/org/eclipse/xtext/builder/trace/TraceEditor.java]: Fixed.

File: org.eclipse.xtext.builder/src/org/eclipse/xtext/builder/trace/TraceEditor.java (hunk 1 above)
Comment (ArneDeutsch) [reply org.eclipse.xtext.builder/src/org/eclipse/xtext/builder/trace/TraceEditor.java]: Fixed on MacOS. Modifier ist not shown on Windows/Linux. Probably an issue with with eclipse shortcut system. But hotkey works, just a minor display issue.

**Review by cdietrich — APPROVED** [review]

**Pull request merged**